Source/OxyPlot/PlotModel/PlotModel.cs

```diff
@@ -193,6 +193,16 @@ public enum TitleHorizontalAlignment
     public partial class PlotModel : Model, IPlotModel
     {
         /// <summary>
+        /// The snap timer.
+        /// </summary>
+        private readonly Timer snapTimer;
```

Comment:
I know, I wish there was another (easy) way. But because the timer is inside the model, not inside the control, it will also work during printing. And... the timer will only kick in when you pan / zoom / navigate anyway.